lib/ipmi_backend_utils.pm

```diff
@@ -207,7 +207,7 @@ sub get_installation_partition {
 
 sub set_pxe_efiboot {
     my ($root_prefix) = @_;
-    $root_prefix //= "/";
+    $root_prefix = "/" if ($root_prefix eq "") || (!defined $root_prefix);
```

Comment:
Even though there is no built in thing to do exactly what you have written, but `||=` will solve the issue, except that setting `$root_prefix` to 0 will trigger it too. Otherwise, I would at least flip conditions, as in case `$root_prefix` is undef, we will check both conditions, but with different order we will return after checking first condition. That's minor, but will work faster =).

Reply:
@rwx788 
Do you mean 
$root_prefix = "/" if (!defined $root_prefix) || ($root_prefix eq "");
works faster ? 
Can I apply this change ?

Reply:
@rwx788 Would you please help to confirm?